### PR TITLE
Subscriptions

### PR DESCRIPTION
**Pull request by clone1018**

### CODE REVIEW

File: lib/glimesh_web/controllers/user_payments_controller.ex [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    )
+  end
+
+  def connect(conn, %{"state" => state, "code" => code}) do
+    user = conn.assigns.current_user
+    #    Plug.CSRFProtection.valid_state_and_csrf_token?(state)
+    case Glimesh.Payments.oauth_connect(user, code) do
+      {:ok, _} ->
+        conn
+        |> put_flash(:info, "Stripe account linked successfully, welcome to the sub club!")
```
Comment (mja00) [inline lib/glimesh_web/controllers/user_payments_controller.ex:39]: Gettext pls :)

File: lib/glimesh_web/controllers/user_payments_controller.ex [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    end
+  end
+
+  def delete_default_payment(conn, %{}) do
+    user = conn.assigns.current_user
+
+    case Glimesh.Accounts.set_stripe_default_payment(user, nil) do
+      {:ok, _} ->
+        conn
+        |> put_flash(:info, "Payment method deleted!")
```
Comment (mja00) [inline lib/glimesh_web/controllers/user_payments_controller.ex:55]: Also gettext :)

File: lib/glimesh_web/controllers/user_payments_controller.ex (hunk 1 above)
Comment (clone1018) [reply lib/glimesh_web/controllers/user_payments_controller.ex:39]: How to?

File: lib/glimesh_web/live/platform_subscription_live/index.html.leex [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        </div>
+    </div>
+
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-8">
+                <div class="pricing-plans-container mt-5 d-md-flex d-block">
+                    <!-- Plan -->
+                    <div class="pricing-plan mb-5">
+                        <h3>Lil Meshi</h3>
```
Comment (mja00) [inline lib/glimesh_web/live/platform_subscription_live/index.html.leex:17]: You can probably guess with this one too :)

**Pull request merged**